src/app/static/src/js/components/runWorkflow/workflowSummary/runWorkflowSummary.vue

```diff
@@ -1,39 +1,46 @@
 <template>
     <div id="summary-header">
+        <runWorflowSummaryHeader :workflow-summary="workflowSummary"></runWorflowSummaryHeader>
```

Comment:
this has a spelling error and should be kebab cased: `run-workflow-summary-header`

Reply:
Me too! I thought at last the linter would complain!